--- NTLconvert.cc.orig	Thu Oct 10 19:43:38 2002
+++ NTLconvert.cc	Thu Mar 27 06:37:24 2003
@@ -25,6 +25,8 @@
 #include <NTL/ZZ_pEXFactoring.h>
 #include <NTL/GF2EXFactoring.h>
 #include "NTLconvert.h"
+#include <string.h>		// strcat
+#include "memutil.h"		// getBlock, freeBlock
 
 ////////////////////////////////////////////////////////////////////////////////
 // NAME: convertFacCF2NTLZZpX                                                 //
@@ -526,10 +528,10 @@ ZZX convertFacCF2NTLZZX(CanonicalForm f)
 
         gmp_val[0]=getmpi(i.coeff().getval());
         int l=mpz_sizeinbase(gmp_val,10)+2;
-        stringtemp=(char*)omAlloc(l);
+        stringtemp=(char*)getBlock(l);
         stringtemp=mpz_get_str(stringtemp,10,gmp_val);
         conv(temp,stringtemp);
-        omFreeSize(stringtemp,l);
+        freeBlock(stringtemp,l);
 
         //set the computed coefficient
         SetCoeff(ntl_poly,NTLcurrentExp,temp);
--- NTLconvert.h.orig	Tue Jul 30 17:20:02 2002
+++ NTLconvert.h	Thu Mar 27 05:46:44 2003
@@ -27,6 +27,7 @@
 #include <limits.h>
 #include <NTL/ZZ_pEXFactoring.h>
 #include <NTL/GF2EXFactoring.h>
+using namespace NTL;
 
 ZZ_pX convertFacCF2NTLZZpX(CanonicalForm f);
 GF2X convertFacCF2NTLGF2X(CanonicalForm f);
