--- factor/version.cc.orig	Wed Aug 22 16:35:27 2001
+++ factor/version.cc	Sun Mar 30 05:21:14 2003
@@ -2,12 +2,12 @@
 ///////////////////////////////////////////////////////////////////////////////
 // emacs edit mode for this file is -*- C++ -*-
 // $Id$
-static char * rcsid = "@(#) libfac version 0.2.4 [May 29 1997] (C) 1996-97 M. Messollen";
+static char * rcsid = "@(#) libfac version 2.0.4 [Nov 7 2002] (C) 1996, 1997, 2002 M. Messollen";
 ///////////////////////////////////////////////////////////////////////////////
 
 const char * libfac_name = "Factorization and characteristic sets library";
-const char * libfac_version = "0.3.3";
-const char * libfac_date = "Aug 22 2001";
+const char * libfac_version = "2.0.4";
+const char * libfac_date = "Nov 7 2002";
 const char * libfac_author = "M. Messollen";
 
 int    libfac_interruptflag=0;
