--- configure.orig	Wed Aug 22 16:36:37 2001
+++ configure	Sun Mar 30 05:13:37 2003
@@ -533,9 +533,9 @@
 
 libfac_name="\"Factorization and characteristic sets library\""
 
-libfac_version="0.3.3"
+libfac_version="2.0.4"
 
-libfac_date="\"Aug 22 2001\""
+libfac_date="\"Nov 7 2002\""
 
 libfac_author="\"M. Messollen\""
 
@@ -965,120 +965,120 @@
 
 
 
-echo $ac_n "checking for working const""... $ac_c" 1>&6
-echo "configure:970: checking for working const" >&5
-if eval "test \"`echo '$''{'ac_cv_c_const'+set}'`\" = set"; then
-  echo $ac_n "(cached) $ac_c" 1>&6
-else
-  cat > conftest.$ac_ext <<EOF
-#line 975 "configure"
-#include "confdefs.h"
 
-int main() {
 
-/* Ultrix mips cc rejects this.  */
-typedef int charset[2]; const charset x;
-/* SunOS 4.1.1 cc rejects this.  */
-char const *const *ccp;
-char **p;
-/* NEC SVR4.0.2 mips cc rejects this.  */
-struct point {int x, y;};
-static struct point const zero = {0,0};
-/* AIX XL C 1.02.0.0 rejects this.
-   It does not let you subtract one const X* pointer from another in an arm
-   of an if-expression whose if-part is not a constant expression */
-const char *g = "string";
-ccp = &g + (g ? g-g : 0);
-/* HPUX 7.0 cc rejects these. */
-++ccp;
-p = (char**) ccp;
-ccp = (char const *const *) p;
-{ /* SCO 3.2v4 cc rejects this.  */
-  char *t;
-  char const *s = 0 ? (char *) 0 : (char const *) 0;
-
-  *t++ = 0;
-}
-{ /* Someone thinks the Sun supposedly-ANSI compiler will reject this.  */
-  int x[] = {25, 17};
-  const int *foo = &x[0];
-  ++foo;
-}
-{ /* Sun SC1.0 ANSI compiler rejects this -- but not the above. */
-  typedef const int *iptr;
-  iptr p = 0;
-  ++p;
-}
-{ /* AIX XL C 1.02.0.0 rejects this saying
-     "k.c", line 2.27: 1506-025 (S) Operand must be a modifiable lvalue. */
-  struct s { int j; const int *ap[3]; };
-  struct s *b; b->j = 5;
-}
-{ /* ULTRIX-32 V3.1 (Rev 9) vcc rejects this */
-  const int foo = 10;
-}
 
-; return 0; }
-EOF
-if { (eval echo configure:1024: \"$ac_compile\") 1>&5; (eval $ac_compile) 2>&5; }; then
-  rm -rf conftest*
-  ac_cv_c_const=yes
-else
-  echo "configure: failed program was:" >&5
-  cat conftest.$ac_ext >&5
-  rm -rf conftest*
-  ac_cv_c_const=no
-fi
-rm -f conftest*
-fi
 
-echo "$ac_t""$ac_cv_c_const" 1>&6
-if test $ac_cv_c_const = no; then
-  cat >> confdefs.h <<\EOF
-#define const 
-EOF
 
-fi
 
-echo $ac_n "checking for inline""... $ac_c" 1>&6
-echo "configure:1045: checking for inline" >&5
-if eval "test \"`echo '$''{'ac_cv_c_inline'+set}'`\" = set"; then
-  echo $ac_n "(cached) $ac_c" 1>&6
-else
-  ac_cv_c_inline=no
-for ac_kw in inline __inline__ __inline; do
-  cat > conftest.$ac_ext <<EOF
-#line 1052 "configure"
-#include "confdefs.h"
 
-int main() {
-} $ac_kw foo() {
-; return 0; }
-EOF
-if { (eval echo configure:1059: \"$ac_compile\") 1>&5; (eval $ac_compile) 2>&5; }; then
-  rm -rf conftest*
-  ac_cv_c_inline=$ac_kw; break
-else
-  echo "configure: failed program was:" >&5
-  cat conftest.$ac_ext >&5
-fi
-rm -f conftest*
-done
 
-fi
 
-echo "$ac_t""$ac_cv_c_inline" 1>&6
-case "$ac_cv_c_inline" in
-  inline | yes) ;;
-  no) cat >> confdefs.h <<\EOF
-#define inline 
-EOF
- ;;
-  *)  cat >> confdefs.h <<EOF
-#define inline $ac_cv_c_inline
-EOF
- ;;
-esac
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 
 
 if test ${GXX} = yes; then
@@ -1089,7 +1089,7 @@
     else
        CXXFLAGS="-O3 "
     fi
-# check whether CXX accepts --no-rtti
+# check whether CXX accepts -fno-rtti
 
 ac_ext=C
 # CXXFLAGS is not in ac_cpp because -g, -O, etc. are not valid cpp options.
@@ -1098,10 +1098,10 @@
 ac_link='${CXX-g++} -o conftest${ac_exeext} $CXXFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ext $LIBS 1>&5'
 cross_compiling=$ac_cv_prog_cxx_cross
 
-echo $ac_n "checking whether gcc accepts --no-rtti""... $ac_c" 1>&6
-echo "configure:1103: checking whether gcc accepts --no-rtti" >&5
+echo $ac_n "checking whether gcc accepts -fno-rtti""... $ac_c" 1>&6
+echo "configure:1103: checking whether gcc accepts -fno-rtti" >&5
 tmp_flags=${CXXFLAGS}
-CXXFLAGS="${CXXFLAGS} --no-rtti"
+CXXFLAGS="${CXXFLAGS} -fno-rtti"
 if eval "test \"`echo '$''{'ac_cv_cxx_have_rtti'+set}'`\" = set"; then
   echo $ac_n "(cached) $ac_c" 1>&6
 else
@@ -1129,13 +1129,13 @@
 echo "$ac_t""${ac_cv_cxx_have_rtti}" 1>&6
 CXXFLAGS=$tmp_flags
 if test "${ac_cv_cxx_have_rtti}" = yes; then
-CXXFLAGS="$CXXFLAGS --no-rtti"
+CXXFLAGS="$CXXFLAGS -fno-rtti"
 fi
 
-echo $ac_n "checking whether gcc accepts --no-exceptions""... $ac_c" 1>&6
-echo "configure:1137: checking whether gcc accepts --no-exceptions" >&5
+echo $ac_n "checking whether gcc accepts -fno-exceptions""... $ac_c" 1>&6
+echo "configure:1137: checking whether gcc accepts -fno-exceptions" >&5
 tmp_flags=${CXXFLAGS}
-CXXFLAGS="${CXXFLAGS} --no-exceptions"
+CXXFLAGS="${CXXFLAGS} -fno-exceptions"
 if eval "test \"`echo '$''{'ac_cv_cxx_have_exceptions'+set}'`\" = set"; then
   echo $ac_n "(cached) $ac_c" 1>&6
 else
@@ -1163,7 +1163,7 @@
 echo "$ac_t""${ac_cv_cxx_have_exceptions}" 1>&6
 CXXFLAGS=$tmp_flags
 if test "${ac_cv_cxx_have_exceptions}" = yes; then
-CXXFLAGS="$CXXFLAGS --no-exceptions"
+CXXFLAGS="$CXXFLAGS -fno-exceptions"
 fi
 
 
