--- factor/tmpl_inst.cc.orig	Wed Aug  8 13:59:13 2001
+++ factor/tmpl_inst.cc	Thu Mar 27 19:25:51 2003
@@ -80,12 +80,12 @@
 #ifndef NOSTREAMIO
 template ostream & operator << ( ostream &, const List<CFList> & );
 template ostream & operator << ( ostream &, const List<Variable> & );
-template ostream & operator << ( ostream &, const List<int> & );
 template ostream & operator << ( ostream &, const Array<int> & );
 #endif
 
 template class Array<int>;
 template class Array<Variable>;
+template class Array<REvaluation>;
 
 // for database
 #ifdef HAVE_DATABASE
