--- configure.ac.orig	Wed Jan  9 17:21:34 2002
+++ configure.ac	Fri Mar 28 02:38:32 2003
@@ -169,22 +169,26 @@
 AC_CHECK_FUNC(__isinf,,AC_DEFINE(__isinf,isinf,whether __isinf is missing))
 
 OLIBS="$LIBS"
-LIBS="$LIBS -lgmp -lgc -lgdbm -lcf -lfac"
+LIBS="-lgc $OLIBS"
 AC_TRY_LINK(
     [#include <gc.h>],
     [GC_malloc(1);],
     ,AC_MSG_WARN(missing library: libgc))
+LIBS="-lgmp $OLIBS"
 AC_TRY_LINK(
     [#include <gmp.h>],
     [__gmp_version;],
     ,AC_MSG_WARN(missing library: libgmp))
+LIBS="-lgdbm $OLIBS"
 AC_TRY_LINK([#include <gdbm.h>],
     [gdbm_close(0);],
     ,AC_MSG_WARN(missing library: libgdbm))
+LIBS="-lcf -lcfmem -lgmp $OLIBS"
 AC_TRY_LINK(
     [#include <factoryconf.h>],
     [FACTORYVERSION;],
-    ,AC_MSG_WARN(missing library: libcf))
+    ,AC_MSG_WARN(missing library: libcf libcfmem))
+LIBS="-lfac -lcf -lcfmem -lgmp $OLIBS"
 AC_TRY_LINK(
     [extern const char * libfac_version;],
     [libfac_version;],
